src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<p>1.</p><blockquote>or modify the existing functions, which <strong>is </strong>strongly advised against</blockquote><p>Besides the typo, I don't really understand the rationale here. Could you elaborate please ?</p><p> </p><p>2.</p><p><code>mtow_loop_and_fuel </code>should be renamed <code>mtow_and_fuel_loop</code> (both file name and function... and all occurences in the notebook)</p><p> </p><p>3.</p><p>More importantly: though I find nice to address the question of new variables, it looks a bit artificial to me, since you have litteraly the consumed fuel variable in the performance model. As I stated in the previous notebook, the problem is that MTOW is an output of the performance model.</p><p> </p><p>We may keep things as they are now, but then I would advise to clearly tell that we purposely arranged models that way.</p><p> </p><p>But if possible, I'd rather have a first optimization on fuel (is the optimization of MTOW really useful?) and add a really new variable that could be optimized (though I do not have any idea right now).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
1. has been deleted
2. Corrected in 4d319923
3. Corrective action discussed during meeting and implemented

Reply:
<p>You still have 2 occurrences of <code>mtow_loop_and_fuel</code> (alink and its associated text) in "Optimization with two parameters"</p>

Reply:
<p>Corrected in 7a6643b4, sorry for the oversight</p>